bcgov/PIMS
# DEVOP-155 Switch from truncating tickets to sending two requests

## 🎯 Summary

  
[DEVOP-155: ](https://citz-imb.atlassian.net/browse/DEVOP-155)


Updating the script such that instead of truncating and not posting tickets we send multiple requests.

To test changes:
1. Navigate to Actions tab on PIMS repo. 
2. on left menu select "App NPM Dependency Updates
3. select the dropdown "Run Workflow" 
    1. change branch to run on to "DEVOP-155-remove-truncation"
    2. select "Run workflow"
4. Once all tasks are complete check PIMS backlog
    - https://citz-imb.atlassian.net/jira/software/c/projects/PIMS/boards/26/backlog
    - at the very bottom there should be a new parent ticket "Dependency Updates - DATE"
    - The sub tasks under that parent ticket should reflect all minor and major dependency updates that are not currently in an open ticket on the PIMS board
        - As of Oct 27th at 8:30am ish the only ticket it should create is to update axios. This may change as new updates are released.


## 🔰 Checklist


> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.
